src/controllers/message.controller.js

```diff
@@ -0,0 +1,36 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+const MESSAGE = 'message';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+
+  if (!text || !author || !roomId) {
+    res.status(400).send({ message: 'one of nessesery parys is required' });
+  }
```

Comment:
The error message in line 20 is not very informative. It would be better to specify which parameters are missing in the request. This will help the client to understand what exactly went wrong.